doc/api/fs.md

```diff
@@ -1872,10 +1879,16 @@ object with an `encoding` property specifying the character encoding to use for
 the path passed to the callback. If the `encoding` is set to `'buffer'`,
 the path returned will be passed as a `Buffer` object.
 
+*Note*: If `path` resolves to a Socket or a Pipe, the function will return the
```

Comment:
No I hurry copy-pasted your comment, removed the both as I couldn't reach to anything useful for here.